Added support for MS SQL server driver for PHP 7.0

markhilton opened the pull request:
compiled extension for sqlsrv.so driver for the workspace container

winfried-van-loon commented on workspace/Dockerfile-70:
```diff
@@ -394,6 +394,85 @@ RUN if [ ${INSTALL_LINUXBREW} = true ]; then \
 #####################################
+# SQL SERVER:
+#####################################
+ARG INSTALL_MSSQL=true
```
The `INSTALL_MSSQL` looks great here. It should be added on the following places to integrate it into Laradock:

- `docker-compose.yml` in the `workspace` > `build` > `args` section.
- `env-sample` in the `Containers Customization` > `WORKSPACE` section.

winfried-van-loon commented:
Also, before merging, we may want to upgrade the sqlsrv version as mentioned in #843 to avoid that error recurring. This is related to #882 and #883.

markhilton commented:
If I remember well I added:

cd / && \

because originally I added MS SQL support build after WORKDIR inside Dockerfile, therefore wget was downloading unixodbc libs into my application code base, as workdir is a volume mounted to local filesystem. It can be removed. I will clean this up and re-submit.

markhilton commented:
I submitted updates for this pull requests but seems like build on Travis fails due to unrelated issue with missing XML for INSTALL_V8JS_EXTENSION

winfried-van-loon commented:
Yep, that's open issue #792, not quite sure what to do about that, to be honest. Thanks for the updates to the pull request, will look at your changes very soon.

winfried-van-loon commented on php-fpm/Dockerfile-70:
```diff
@@ -312,6 +312,81 @@ RUN if [ ${INSTALL_GHOSTSCRIPT} = true ]; then \
+    #####################################
+
+    pecl install sqlsrv-4.1.7 && \
+    pecl install pdo_sqlsrv-4.1.7 && \
```
Version of these should be increased

winfried-van-loon commented on workspace/Dockerfile-70:
```diff
@@ -508,6 +508,81 @@ RUN if [ ${INSTALL_SYMFONY} = true ]; then \
+    #####################################
+
+    pecl install sqlsrv-4.1.7 && \
+    pecl install pdo_sqlsrv-4.1.7 && \
```
Version of these should be increased

winfried-van-loon commented on php-fpm/Dockerfile-70:
```diff
@@ -273,8 +273,8 @@ RUN if [ ${MSSQL} = true ]; then \
-    pecl install sqlsrv-4.0.6 && \
-    pecl install pdo_sqlsrv-4.0.6 && \
+    pecl install sqlsrv-4.1.7preview && \
+    pecl install pdo_sqlsrv-4.1.7preview && \
```
Version number change reference

winfried-van-loon replied:
Or we should lower this to stable, now that I'm looking at it.
